runtime/libpgmath/lib/generic/qcos.c

```diff
@@ -0,0 +1,14 @@
+/*
+ * Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+ * See https://llvm.org/LICENSE.txt for license information.
+ * SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+ *
+ */
+
+/* Intrinsic function which take quad precision arguments. */
+
+#include "mthdecls.h"
+long double __mth_i_qcos(long double d)
```

Comment:
empty line (as in other files) would help with readibility.

Reply:
Fixed.